spec/unit/provider/group/windows_adsi_spec.rb

```diff
@@ -151,7 +173,8 @@
 
       create = sequence('create')
       group.expects(:commit).in_sequence(create)
-      group.expects(:set_members).with(['user1', 'user2'], true).in_sequence(create)
+      # it seems a bit strange that defaultto false returns nil
+      group.expects(:set_members).with(['user1', 'user2'], nil).in_sequence(create)
```

Comment:
It appears puppet does not handle default false values correctly due to a long standing bug PUP-1967. In https://github.com/puppetlabs/puppet/blob/3.7.3/lib/puppet/type.rb#L813-L817, if the default value is `false` then it's not truthy, so `value = parameter.default` is false, preventing the default `parameter.value` from being initialized correctly.